evaluations/data/evaluation_configs.py

```diff
@@ -77,10 +77,131 @@ def _smoke_test(num_runs=100):
           )
       )
 
+
+def _test_with_selected_parameters_second_part(num_runs=100):
+  """Configurations with handy selected parameters for scenarios 3b - 5.
+
+  Args:
+    num_runs: the number of runs per scenario * parameter setting.
+
+  Returns:
+    An EvaluationConfig.
+  """
+  scenario_config_list = []
+  # Common parameters
+  universe_size = 100000
+  # Now fix the universe size. Can vary it if necessary.
+  num_sets = 20
+  order = set_generator.ORDER_RANDOM
+  # Apply random order of sets here, assuming that we shuffle the publishers
+  # before the estimation.
+  small_reach_rate = 0.01
+  large_reach_rate = 0.2
+  small_reach = int(small_reach_rate * universe_size)
+  large_reach = int(large_reach_rate * universe_size)
+
+  # Scenario 3 (b). Exponential bow, identical user behavior
```

Comment:
It might be slightly more readable to pull this into its own function, e.g.,
  def _generate_scenario_3b_exponential_bow_identical_users(...):
    ...

Reply:
Done.